gws-client-spring-boot-starter/src/main/java/edu/uw/edm/gws/autoconfigure/GWSAutoConfiguration.java

```diff
@@ -28,8 +53,63 @@ public GWSAutoConfiguration(GWSProperties gwsProperties) {
 
     @Bean
     @ConditionalOnMissingBean
-    public GroupsWebServiceClient groupsWebServiceFacade(RestTemplate restTemplate) {
+    public GroupsWebServiceClient groupsWebServiceFacade(@Qualifier("gws-client") RestTemplate restTemplate) {
 
         return new GroupsWebServiceClientImpl(restTemplate, gwsProperties.getUrl());
     }
+
+    private CloseableHttpClient httpClient(final PoolingHttpClientConnectionManager connectionManager) throws NoSuchAlgorithmException, KeyManagementException {
+
+        RequestConfig.Builder requestConfigBuilder = RequestConfig.custom().setExpectContinueEnabled(true);
```

Comment:
Why are you returning the builder rather than RequestConfig, it is never used as a builder.